plugin/markdown/plugin.js

```diff
@@ -415,6 +428,73 @@ const Plugin = () => {
 
 	}
 
+	function parseFrontMatter(content, options) {
+		options = getSlidifyOptions( options)
+
+		const parsedFrontMatter = fm(content)
+
+		content = parsedFrontMatter.body;
+		if (parsedFrontMatter.frontmatter){
+			options.metadata = yaml.load(parsedFrontMatter.frontmatter);
+			if (!('slideType' in options.metadata)) {
+				content = `Missing "slideType" in default metadata`
+				console.error(content)
+				delete options.metadata
+			}
+		}
+		return [content, options];
+	}
+
+	function parseMarkdown(markdown, options) {
```

Comment:
I think this function is about separating the yaml from the markdown for a single slide. If that is true it should be named accordingly. `parseMarkdown` sounds very generic to me

Reply:
I have changed the name of a function to `separateInlineMetadataAndMarkdown`. Does that sound better?